pkg/virt-controller/services/multus_annotations_test.go

```diff
@@ -0,0 +1,100 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package services
```

Comment:
Testing private functions is coupling the tests to the internal implementation and that may be worse than having none.
I'm unclear about what can be done in this specific case, I would just try to examine if these tests can be checked from the formal API (public functions).

Reply:
As I see it, 2 choices here: either get this in as is (with these unit tests, that check the un-exported functions), or figure that out in a follow up PR. 

Multus annotation generation is already tested by the template generation, in https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-controller/services/template_test.go#L715

Reply:
So are those tests cover the scenarios you presented here?

Reply:
More than that.

Reply:
Too short answer form me to understand.
If these scenarios are already covered, I do not understand why you created a duplication now.

Ideally, if you can check these scenarios through the exported functions, that is preferred. If that means to move the existing tests that are relevant to here, great.
I just ask you to check the options of this, if you see it is too "heavy" then we can defer it to later.

Reply:
Let's defer to later then.